core/frontend/src/test/render/primitives/MeshBuilder.test.ts

```diff
@@ -137,19 +137,19 @@ describe("Mesh Builder Tests", () => {
 
     const loopRange: Range3d = new Range3d();
     loop.range(undefined, loopRange);
-    expect(loopRange).to.not.be.null;
+    expect(loopRange.isNull).toBe(false);
```

Comment:
i will be honest, many of places where this was done was me spamming copilot to refactor and move on 🥲 I can go back and do a 2nd refactor, it should be easier since copilot (hopefully) is consistent with where it does all this